examples/dsp/cmd_reply/arm7/source/main.c

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: CC0-1.0
```

Comment:
I can probably simplify this example so that it only has code for ARM9 and the DSP, but I think eventually someone will need a template with the 3 CPUs.

Reply:
Absolutely. I expect the kind of person to reach for the DSP (and not through some kind of eventually-well-established music library) to want to target all three CPUs.